### PR TITLE
Simplifying steps for GHA tests using "poor-man's ternary"

### PR DESCRIPTION
**Pull request by fletch3555**

### CODE REVIEW

**Comment by ArlonAntonius** [comment]
Looks like this does not work? Non of the test jobs run now?

**Comment by ArlonAntonius** [comment]
I don't quite understand why the tests are not running, but that worries me. Could you take a look at that tomorrow?

**Comment by fletch3555** [comment]
it's like the test.yml file doesn't exist.... wtf

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -76,13 +79,12 @@ jobs:
...
 
-            - name: Install Version Specific Dependencies
-              if: ${{ ! matrix.experimental }}
-              run: composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
-
-            - name: Install Version Specific Dependencies (ignore-platform-req)
+            - name: Setup Composer args
               if: ${{ matrix.experimental }}
-              run: composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update --ignore-platform-reqs
+              run: echo "COMPOSER_ARGS=--ignore-platform-reqs" >> $GITHUB_ENV
```
Comment (ArlonAntonius) [inline .github/workflows/test.yml:84]: This looks really weird, isnt there an easier way?

**Review by ArlonAntonius — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (fletch3555) [reply .github/workflows/test.yml:84]: Agreed, but nope, that's the official mechanism for doing it... https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

**Pull request merged**